test/test_va_api_fixture.cpp

```diff
@@ -94,13 +96,33 @@ static VADisplay getDrmDisplay(int &fd)
     typedef std::vector<std::string> DevicePaths;
     typedef DevicePaths::const_iterator DevicePathsIterator;
 
-    static DevicePaths paths({"/dev/dri/renderD128", "/dev/dri/card0"});
+    static DevicePaths paths({
+        "/dev/dri/renderD128",
+        "/dev/dri/card0",
+        "/dev/dri/renderD129",
+        "/dev/dri/card1",
```

Comment:
Instead of using hard-coded paths, can we use the `drmGetDevices2` API - see the `tests/drmdevice.c` from the libdrm repo.

Reply:
Thanks for the tip. Done.